doc/whats_new/v0.22.rst

```diff
@@ -419,7 +419,7 @@ Changelog
 ....................
 
 - |Fix| :class:`dummy.DummyClassifier` now handles checking the existence
-  of the provided constant in multiouput cases.
+  of the provided constant in multioutput cases.
```

Comment:
[CI fails](https://app.circleci.com/jobs/github/scikit-learn/scikit-learn/190617?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) because of another issue when changing this file. For now, you revert this change?